src/utils/create-get-visitor-keys.js

```diff
@@ -7,7 +7,7 @@ function createGetVisitorKeys(visitorKeys, typeProperty = "type") {
     const type = node[typeProperty];
 
     /* c8 ignore next 5 */
-    if (process.env.NODE_ENV !== "production" && type === undefined) {
+    if (type === undefined && process.env.NODE_ENV !== "production") {
```

Comment:
It's intentionally write like this, since `process.env.NODE_ENV !== "production"` will be `false` after we bundled.

Reply:
And this branch will be removed in published version.

Reply:
Oh, I see. I should have spent more time looking into the build process. 😉
Thanks for the prompt review.